src/main/java/org/jenkinsci/plugins/ewm/actions/ExwsAllocateActionWrapper.java

```diff
@@ -0,0 +1,51 @@
+package org.jenkinsci.plugins.ewm.actions;
+
+import hudson.model.Actionable;
+import hudson.model.Run;
+import jenkins.model.RunAction2;
+import org.jenkinsci.plugins.ewm.Messages;
+
+/**
+ * A {@link RunAction2} that acts as a wrapper to hold multiple {@link ExwsAllocateActionImpl}s.
+ *
+ * @author Alexandru Somai
+ */
+public class ExwsAllocateActionWrapper extends Actionable implements RunAction2 {
+
+    private Run parent;
+
+    public Run getParent() {
```

Comment:
No, it's fine then. Just add <code>@Restricted(NoExternalUse)</code> to prevent the wide exposure